packages/datadog-instrumentations/src/helpers/hooks.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
   'jest-config': () => require('../jest'),
   'jest-environment-node': () => require('../jest'),
   'jest-environment-jsdom': () => require('../jest'),
+  '@jest/test-sequencer': () => require('../jest'),
```

Comment:
This should be in alphabetical order.

Reply:
good catch. Fixed in https://github.com/DataDog/dd-trace-js/pull/3452/commits/02f357f6edc17df2f43d5fab10d02b0f7bc763c2